server/app/scanpy_engine/scanpy_engine.py

```diff
@@ -40,27 +42,28 @@ def _alias_annotation_names(self, axis, name):
         As a *critical* side-effect, ensure the indices are simple number ranges
         (accomplished by calling pandas.DataFrame.reset_index())
         """
-        if name == 'name':
+        if name == "name":
             # a noop, so skip it
             return
 
         ax_name = str(axis)
         df_axis = getattr(self.data, ax_name)
         if name is None:
-            # reset index to simple range; alias 'name' to point at the
+            # reset index to simple range; alias "name" to point at the
             # previously specified index.
-            df_axis = df_axis.reset_index().rename(columns={'index': 'name'})
+            df_axis.reset_index(inplace=True)
```

Comment:
Just want to confirm you have validated the performance implications of using `inplace`?  (I don't know of any issues, just think it is worth checking)

Reply:
Speeds it up actually:
Master: TIME 0.2940518856048584
This Branch: TIME: 0.04870891571044922

Reply:
⬆️ is for the 1.3 mil dataset

Reply:
nice!